.guardian.yml

```diff
@@ -1,5 +1,5 @@
-files: ./shard.yml
-run: shards install
----
 files: ./**/*.cr
-run: crystal spec
+run: crystal build ./src/readability.cr
+---
+files: ./shard.yml
+run: crystal deps
```

Comment:
Ouch, `crystal deps` got removed, better to change back to `shards install`.

Reply:
Hm yeah what the hell happened there. Ah well I'll fix in master, thanks for noticing 👍